components/src/molecules/file-upload/FileUpload.html

```diff
@@ -0,0 +1,60 @@
+<div class="file-upload">
+  <input
+    :class="['file-upload__input', inputClass]"
+    :id="inputId"
+    :name="inputName"
+    type="file"
+    ref="fileInput"
+    :accept="acceptedFormats"
+    @change="updateFile"
+  >
+  <a-label
```

Comment:
I'm wondering if it's correct to wrap whole content to label tag?

Reply:
I think it's ok to use label as a wrapper to some texts and a button

Reply:
let's keep this shape for now